internal/kibana/synthetics/monitor_http/schema.go

```diff
@@ -0,0 +1,72 @@
+package monitor_http
+
+import (
+	"context"
+	"github.com/elastic/terraform-provider-elasticstack/internal/kibana/synthetics"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+)
+
+//TODO:
+// Ensure provider defined types fully satisfy framework interfaces
+
+type Resource struct {
+	//client *clients.ApiClient
+}
+
+func HttpMonitorModeSchema() schema.Attribute {
```

Comment:
Does it make sense to leave this out and focus on the private location resource in this PR?